### DISCUSSION TITLE
20211019/20211020 release

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#571] [Bug Reports]
@rern 

I see that you have released an update (20211019) earlier this morning however it doesn't seem to be available to apply via the web app as yet. Do you need to do something else to make it available ?

Thanks

Chris

**Comment by ceejayemm** [comment]
Update seems to have now been released. 

Applied 20211019 update to RPi3B+ (i20211015, 64bit base) but now Networks screen not available due to JSON error:

Errors: Unexpected token } in JSON at position 341
{ "page" : "networks" , "activebt" : false, "activeeth" : true , "activewlan" : true , "listbt" : false, "listeth" : false, "listwl" : [ { "dbm" : -58 , "gateway" : "192.168.86.1" , "hidden" : false, "hostname" : "raconservatory.lan" , "ip" : "192.168.86.33" , "password" : "XXXXXXX" , "ssid" : "CMHomeNET" , "static" : false , "wep" : ▮} ] , "hostapd" : false, "hostname" : "RAConservatory"}

Chris

**Reply by ceejayemm** [reply]
This JSON error seems to have been fixed with 20211020.

Chris

**Comment by ceejayemm** [comment]
And another JSON error. This occurs after turning on the Bluetooth Equalizer and returning to the main page:

Error: Unexpected token / in JSON at position 450
{ "player" : "mpd" , "btclient" : true , "consume" : false , "control" : "" , "counts" : { "album": 316, "albumartist": 126, "artist": 563, "composer": 1007, "conductor": 2, "date": 154, "genre": 35, "nas": 1, "sd": 0, "usb": 0, "song": 4768, "webradio": 23 } , "file" : "" , "icon" : "" , "librandom" : false , "playlists" : 0 , "relays" : false , "relayson" : false , "stream" : false , "updateaddons" : false , "updating_db" : false , "volume" : n▮/a , "volumemute" : 0 , "webradio" : false , "repeat" : false , "random" : false , "single" : false , "file" : "http://stream.live.vc.bbcmedia.co.uk/bbc_radio_two" , "playlistlength" : 1 , "song" : 0 , "state" : "play" , "timestamp" : 1634663779847 , "volumemute" : 0 , "stream" : true , "Album" : "" , "Artist" : "BBC Radio 2" , "stationcover" : "/data/webradiosimg/http:||stream.live.vc.bbcmedia.co.uk|bbc_radio_two.1634663779.jpg" , "Name" : "BBC Radio" , "station" : "BBC Radio 2 Stream" , "Time" : false , "Title" : "" , "webradio" : true , "ext" : "Radio" , "coverart" : "" , "icon" : "webradio" , "sampling" : "1/1 • 48.0 kHz 128 kbit/s" , "elapsed" : 129 , "coverart" : ""}

Chris

**Reply by ceejayemm** [reply]
This JSON error still exists after update 20211020. If you turn on the BT Equalizer on the Player screen, go to the Playback screen to view the Equalizer settings Then go back to the Player screen to turn off the Equalizer. Subsequent access to the Playback screen then displays the error.

To get around the error turn off the BT speaker (do not disconnect) and then power on again seems to go through the connection process and fix this error (until the next time you go through the steps above).

Chris

**Comment by ceejayemm** [comment]
New JSON error after update 20211020:

This error occurs after checking out the errors above. The System page gives this JSON error but I am not sure what triggers it.

Errors: Unexpected token ] in JSON at position 1547
{ "page" : "system" , "cpuload" : "0.44 0.38 0.37" , "cputemp" : 47.2 , "startup" : "9s (kernel) + 22s (userspace) " , "throttled" : "0x0" , "time" : "09:57:20 2021-10-20" , "timezone" : "Europe/London" , "uptime" : "16h 52m" , "uptimesince" : "2021-10-19 17:05" , "audioaplayname" : "merus-amp" , "audiooutput" : "MERUS Audio Amp" , "bluetooth" : false, "bluetoothconf" : [ true, false ] , "firmware" : "20211004-1" , "hostapd" : false, "hostname" : "RAConservatory" , "kernel" : "5.10.73-3-raspberrypi4-ARCH aarch64" , "lcd" : false, "lcdchar" : false , "lcdcharaddr" : "0x27 0x3F" , "lcdcharconf" : [ 20,"A00","i2c","0x27","PCF8574",15,18,16,21,22,23,24,false ] , "list" : [ {"icon":"usbdrive","mountpoint":"/boot","mounted":true,"source":"/dev/sda1","size":"50MB/99MB"},{"icon":"usbdrive","mountpoint":"/","mounted":true,"source":"/dev/sda2","size":"2.8GB/7.4GB"},{"icon":"networks","mountpoint":"/mnt/MPD/NAS/NAS_Music","mounted":true,"source":"192.168.86.250:/export/Music/iTunes/Music","size":"693GB/3.6TB"} ] , "lcdmodel" : "tft35a" , "mpdoled" : false , "mpdoledconf" : 6 , "online" : true , "ntp" : "pool.ntp.org" , "powerbutton" : false, "powerbuttonconf" : [ 5,40,5 ] , "relays" : false , "rpimodel" : "Raspberry Pi 3 Model B Plus Rev 1.3" , "soc" : "BCM2837B0" , "soccpu" : "4" , "socram" : "952MiB" , "socspeed" : "1400" , "soundprofile" : true , "soundprofileconf" : [ 145655,60,1000,4000 ] , "version" : "1" , "versionui" : 20211020 , "vuled" : false , "vuledconf" : [ 14,15,18,23,24,25,8 ] , "wlan" : true , "wlanconf" : [ "GB" , ▮] , "wlanconnected" : true}

Chris

**Reply by rern** [reply]
Try:
Addons > rAudio 1 > long-press `Update` > ok for `UPDATE` branch

**Reply by ceejayemm** [reply]
With Update branch applied,  turn on the BT Equalizer on the Player screen, go to the Playback screen to view the Equalizer settings, make no changes to Equalizer settings. Then go back to the Player screen to turn off the Equalizer. Subsequent access to the Playback screen then displays the JSON error.

Error: Unexpected token / in JSON at position 450
{ "player" : "mpd" , "btclient" : true , "consume" : false , "control" : "" , "counts" : { "album": 316, "albumartist": 126, "artist": 563, "composer": 1007, "conductor": 2, "date": 154, "genre": 35, "nas": 1, "sd": 0, "usb": 0, "song": 4768, "webradio": 23 } , "file" : "" , "icon" : "" , "librandom" : false , "playlists" : 0 , "relays" : false , "relayson" : false , "stream" : false , "updateaddons" : false , "updating_db" : false , "volume" : n▮/a , "volumemute" : 0 , "webradio" : false , "repeat" : false , "random" : false , "single" : false , "file" : "http://stream.live.vc.bbcmedia.co.uk/bbc_radio_two" , "playlistlength" : 1 , "song" : 0 , "state" : "play" , "timestamp" : 1634803659084 , "volumemute" : 0 , "stream" : true , "Album" : "" , "Artist" : "BBC Radio 2" , "stationcover" : "/data/webradiosimg/http:||stream.live.vc.bbcmedia.co.uk|bbc_radio_two.1634803658.jpg" , "Name" : "BBC Radio" , "station" : "BBC Radio 2 Stream" , "Time" : false , "Title" : "" , "webradio" : true , "ext" : "Radio" , "coverart" : "" , "icon" : "webradio" , "sampling" : "1/1 • 48.0 kHz 128 kbit/s" , "elapsed" : 22 , "coverart" : ""}

Chris
